lib/OpenQA/Schema/Result/Users.pm

```diff
@@ -63,7 +63,7 @@ __PACKAGE__->add_timestamps;
 __PACKAGE__->set_primary_key('id');
 __PACKAGE__->has_many(api_keys => 'OpenQA::Schema::Result::ApiKeys', 'user_id');
 __PACKAGE__->has_many(
-    development_sessions => 'OpenQA::Schema::Result::DevelopmentSessions',
+    development_sessions => 'OpenQA::Schema::Result::DeveloperSessions',
```

Comment:
I wonder, how is this related? Does it fix something?

Reply:
Yes I saw a warning and decided that it worth to fix it. OK I will make it consistent and will force push as separate commit once all rest is resolved